TestApp/CrashesScreen.js

```diff
@@ -93,14 +104,103 @@ export default class CrashesScreen extends Component {
               Crash native code
             </Text>
           </TouchableOpacity>
+          <TouchableOpacity onPress={() => { this.dialogComponent.show(); }}>
+            <Text style={styles.button}>
+              Set text error attachment
+            </Text>
+          </TouchableOpacity>
+          <Text style={SharedStyles.enabledText}>{'Current value:'}{this.state.textAttachment}</Text>
+          <TouchableOpacity onPress={this.showFilePicker}>
+            <Text style={styles.button}>
+              Select image as binary error attachment
+            </Text>
+          </TouchableOpacity>
+          <Text style={SharedStyles.enabledText}>{'Current value:'}{this.state.binaryAttachment}</Text>
           <Text style={styles.lastSessionHeader}>Last session:</Text>
           <Text style={styles.lastSessionInfo}>
             {this.state.lastSessionStatus}
           </Text>
         </ScrollView>
+        {this.getTextAttachmentDialog()}
       </View>
     );
   }
+
+  getTextAttachmentDialog() {
+    return (
+      <DialogComponent
+        ref={(dialogComponent) => { this.dialogComponent = dialogComponent; }}
+        width={0.9}
+      >
+        <View>
+          <TextInput
+            style={{
+               height: 40, borderColor: 'gray', borderWidth: 1, margin: 8
+              }}
+            onChangeText={text => this.setState({ textAttachment: text })}
+          />
+          <View style={{ flex: 1, flexDirection: 'row', justifyContent: 'space-between' }}>
+            <TouchableOpacity
+              style={{ height: 50 }}
+              onPress={() => {
+                    AttachmentsProvider.saveTextAttachment(this.state.textAttachment);
+                    this.dialogComponent.dismiss();
+              }}
+            >
```

Comment:
[styling]: please use html tag `</TouchableOpacity>` instead of `>` to be consistent with other JSX files in this repo.